### PR TITLE
prep for 3.4.0

### PR DESCRIPTION
**Pull request by nilsnolde**
just misses merging #4066 before releasing 3.4.0 with BETA multimodal support (re-)added

### CODE REVIEW

**Comment by nilsnolde** [comment]
if we wait until next week with the release, we might as well include #4071 and #4075 right? @dnesbitt61 @gknisely would you like to join that review meeting?

**Comment by nilsnolde** [comment]
I suggest to merge this now and release. Other PRs are also interesting to include, but there'll always be those PRs;)

What'ya think @kevinkreiser @dnesbitt61 @gknisely

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-## Release Date: 2023-??-?? Valhalla 3.3.1
+## Release Date: 2022-??-?? Valhalla 3.4.1
+* **Removed**
+* **Bug Fix**
+* **Enhancement**
+
+## Release Date: 2023-04-14 Valhalla 3.4.0
```
Comment (nilsnolde) [inline CHANGELOG.md:6]: just need to remember to change the date of course

**Comment by ImreSamu** [comment]
If time allows, could we also update the `./locales/*` transifex translations?  
As far as I can see, it hasn't been updated yet this year.

And as I see: _"On April 3rd, 2023, we changed our web application's domain, and your organization moved from www.transifex.com/valhalla/ to app.transifex.com/valhalla/."_  (  [changes](https://help.transifex.com/en/articles/7171815-web-application-s-domain-change)  )

**Comment by kevinkreiser** [comment]
i started looking at the localizations and it looked a bit daunting, there were definitely some non trivial things to review.

**Comment by nilsnolde** [comment]
hope in a language we can judge:)

**Comment by nilsnolde** [comment]
I pulled the changes locally and there's only a few language changes for ES, PL, RO and quite a few more for RU. Personally I can't judge any of them properly other than Spanish a bit. Let's take a look later today together.

**Pull request merged**